lib/core/services/auth-guard-base.ts

```diff
@@ -61,7 +62,7 @@ export abstract class AuthGuardBase implements CanActivate, CanActivateChild {
             this.dialog.closeAll();
         }
 
-        return checkLogin;
+        return true;
```

Comment:
correct me if Im wrong, but this means that any other guards extending this base service, with its own 'checkLogin' implementation, [example this one](https://github.com/Alfresco/alfresco-ng2-components/blob/development/lib/core/services/auth-guard-ecm.service.ts#L40), event if it returns `false` wi discard it and return true?